<launch>
   
  <!-- Upload capsule collision operations  & description-->
  <include file="$(find reemc_wbc_tutorials)/launch/reemc_capsule_description_no_head.launch" />
    
  <!-- Configure wbc -->
  <rosparam command="load"
    file="$(find reemc_wbc_tutorials)/config/reemc_wbc_admitance.yaml" />
 
  <!-- Floating base transform publisher -->
  <node name="floating_base_publisher"
        pkg="pal_wbc_controller" type="floating_base_publisher" />

  <!-- Configure local joint control -->
  <rosparam command="load"
    file="$(find reemc_wbc_tutorials)/config/reemc_position_joints.yaml" 
    ns="whole_body_kinematic_controler"/>

  <!-- Spawn controller -->
  <node name="wbc_spawner"
        pkg="controller_manager" type="spawner" output="screen"
        args="whole_body_kinematic_controler" />
        
</launch>
